build.gradle.kts

```diff
@@ -3,8 +3,8 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     id("org.springframework.boot") version "2.1.5.RELEASE"
     id("io.spring.dependency-management") version "1.0.7.RELEASE"
-    kotlin("jvm") version "1.2.71"
-    kotlin("plugin.spring") version "1.2.71"
+    kotlin("jvm") version "1.3.40"
```

Comment:
Actual Kotlin is **1.3.50** iirc.

Reply:
Oh right, intelliJ did want to make me test the 1.3.40 first apparently. Will be updated